compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -1633,7 +1633,7 @@ impl<'a, 'tcx> CrateMetadataRef<'a> {
             if let Some(virtual_dir) = virtual_rust_source_base_dir {
                 if let Some(real_dir) = &sess.real_rust_source_base_dir {
                     if let rustc_span::FileName::Real(old_name) = name {
-                        if let rustc_span::RealFileName::Named(one_path) = old_name {
+                        if let rustc_span::RealFileName::LocalPath(one_path) = old_name {
```

Comment:
The Rust sources should now show up as `rustc_span::RealFileName::Remapped` with their `local_path` being `None`. Unless they were not remapped in the first place, in which case we would not get a match below anyway.